src/main/java/eu/bbmri_eric/negotiator/database/model/Negotiation.java

```diff
@@ -86,7 +84,7 @@ public class Negotiation extends AuditEntity {
   @Exclude
   private Set<PersonNegotiationRole> persons = new HashSet<>();
 
-  @OneToMany(mappedBy = "negotiation", cascade = CascadeType.MERGE, fetch = FetchType.LAZY)
+  @OneToMany(mappedBy = "negotiation", cascade = CascadeType.MERGE, fetch = FetchType.EAGER)
```

Comment:
not sure if eager is needed here.

Reply:
should be removed

Reply:
Fixed in 18f3761c23e6d941a2cd8c7053b98734ce3cd6a8